indico/modules/events/editing/controllers/backend/editable_list.py

```diff
@@ -0,0 +1,57 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2020 CERN
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see the
+# LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from flask import jsonify, request
+
+from indico.modules.events import Event
+from indico.modules.events.contributions.models.contributions import Contribution
+from indico.modules.events.editing.controllers.base import RHEditingBase
+from indico.modules.events.editing.models.editable import EditableType
+
+
+class RHEditableList(RHEditingBase):
+    """Return editable list of the event for the editable type."""
```

Comment:
Maybe you could rephrase this to "Return the list of editables of the event for a given type"